src/app/pages/Privacy.svelte

```diff
@@ -0,0 +1,170 @@
+      -parties of any personal data you can be sure the transfer is legal and
+      secured. These third-party services may have their own privacy policies
+      and we recommend that you review them. They will govern the use of
+      personal information that you submit or which is collected by cookies and
+      other tracking technologies whilst using these services. We do not accept
+      any responsibility or liability for the privacy practices of such third
+      party services and your use of these is at your own risk.
+    </p>
+    <p>
+      We may make changes to this Privacy Policy in the future. You should check
+      this page from time to time to ensure you are aware of any changes. Where
+      appropriate we may notify you of such changes.
+    </p>
+    <h3>2. Information we may collect about you</h3>
+    <p>
+      We do not collect or store any of the data you supply to us while using
+      the Service or website. All data is stored locally on your browser.
+    </p>
+    <h3>3. Collecting, processing and using personal data</h3>
+    <p>We do not store or process any personal data.</p>
+    <h3>4. Why we collect information about you - purpose of processing</h3>
+    <p>
+      We will use information you provide for delivering our Services to you
+      under the terms of use agreed between us. The processing of information in
+      this way is necessary for us to provide you the Service properly and to
+      ensure the features function properly so that you have the best Service
+      possible.
+    </p>
+    <p />
+    <h3>5. Cookies and Web Beacons</h3>
+    <p>We do not use cookies or web beacons.</p>
+    <h3>6. Log files</h3>
+    <p>Log files do not contain any personal data.</p>
+    <p />
+    <h3>7. Children personal data</h3>
+    <p>
+      We do not collect personal information from anyone including those under
+      the age of 18. We encourage parents and legal guardians to monitor their
+      children’s Internet usage and to help enforce this Policy by instructing
+      their children never to provide personal information through the websites
+      or Services.
+    </p>
+    <h3>8. Data sharing</h3>
+    <p>We do not share your data with any third parties.</p>
+    <p>
+      Required by law: We may disclose information to the extent that we are
+      required to do so by law (which may include to government bodies and law
+      enforcement agencies); in connection with any legal proceedings or
+      prospective legal proceedings; and in order to establish, exercise or
+      defend our legal rights.
+    </p>
+    <h3>9. Data transmission</h3>
+    <p>
+      Your data is transmitted safely by using encryption. We secure our website
+      and other systems with technical and organizational measures against the
+      loss, destruction, access, change or dissemination of your data by
+      unauthorized persons.
+    </p>
```

Comment:
If we don't transmit data, then I think this clause is necessary. Regardless, this clause imposes a resonposiblity on the website maintainers (i.e., me), that I'm not uncomfortable guarenteeing 😅  (that said, always use encryption, folks!)

```suggestion
    <h3>9. Data transmission</h3>
    <p>
      Your data is transmitted safely by using encryption. We secure our website
      and other systems with technical and organizational measures against the
      loss, destruction, access, change or dissemination of your data by
      unauthorized persons.
    </p>
```